thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/controller.js

```diff
@@ -168,11 +196,15 @@ export default Controller.extend({
    * @type {Object}
    */
   anomalyStats: computed(
+    'customPercentChange',
+    'selectedSeverityOption',
     'alertEvalMetrics',
     'alertEvalMetrics.projected',
     function() {
       const evalMetrics = this.get('alertEvalMetrics');
-      return buildAnomalyStats(evalMetrics, 'tune');
+      const severity = this.get('customPercentChange');
```

Comment:
You can write this in one line:
`const { evalMetrics, severity } = this.getProperties('alertEvalMetrics', 'customPercentChange')`

Reply:
good destructuring candidate... however, the suggestion is incorrect and will result in undefined vars. Instead...
`const {
        alertEvalMetrics: evalMetrics,
        customPercentChange: severity
      } = this.getProperties('alertEvalMetrics', 'customPercentChange');`

Reply:
Ah, I misread. Yes, I meant your suggestion.